docs/src/reference/attributes.md

```diff
@@ -0,0 +1,232 @@
+The `#[kani::unwind(<number>)]` attribute will:
+ 1. Disable automatic unwinding.
+ 2. Unwind all loops up to `<number>` times.
+
+After the unwinding stage, Kani will attempt to verify the harness.
+If the `#[kani::unwind(<number>)]` attribute was specified, there's a chance that one or more loops weren't unwound enough times.
+In that case, there will be at least one failed unwinding assertion (there's one unwinding assertion for each loop), causing verification to fail.
+
+Check the [*Loops, unwinding and bounds* section](../tutorial-loop-unwinding.md) for more information about stubbing.
+
+### Example
+
+Let's assume we've written this code which contains a loop:
+
+```rust
+fn my_sum(vec: &Vec<u32>) -> u32 {
+    let mut counter = 0;
+    for elem in vec {
+        counter += elem;
+    }
+    counter
+}
+
+#[kani::proof]
+fn my_harness() {
+    let vec = vec![1, 2, 3];
+    let sum = my_sum(&vec);
+    assert!(sum == 6);
+}
+```
+
+Running this example on Kani will produce a successful verification result.
+In this case, Kani automatically finds the required unwinding value (i.e., the number of times it needs to unwind all loops).
+This means that the `#[kani::unwind(<number>)]` attribute isn't needed, as we'll see soon.
+In general, the required unwinding value is equal to the maximum number of iterations for all loops, plus one.
+The required unwinding value in this example is 4: the 3 iterations in the `for elem in vec` loop, plus 1.
+
+Let's see what happens if we force a lower unwinding value with `#[kani::unwind(3)]`:
+
+```rust
+#[kani::proof]
+#[kani::unwind(3)]
+fn my_harness() {
+    let vec = vec![1, 2, 3];
+    let sum = my_sum(&vec);
+    assert!(sum == 6);
+}
+```
+
+As we mentioned, trying to verify this harness causes an unwinding failure:
+
+```
+SUMMARY:
+ ** 1 of 187 failed (186 undetermined)
+Failed Checks: unwinding assertion loop 0
+ File: "/home/ubuntu/devices/src/main.rs", line 32, in my_sum
+
+VERIFICATION:- FAILED
+[Kani] info: Verification output shows one or more unwinding failures.
+[Kani] tip: Consider increasing the unwinding value or disabling `--unwinding-assertions`.
+```
+
+Kani cannot verify the harness because there is at least an unwinding failure.
+But, if we use `#[kani::unwind(4)]`, which is the right unwinding value we computed earlier:
+
+```rust
+#[kani::proof]
+#[kani::unwind(4)]
+fn my_harness() {
+    let vec = vec![1, 2, 3];
+    let sum = my_sum(&vec);
+    assert!(sum == 6);
+}
+```
+
+We'll get a successful result again:
+
+```
+SUMMARY:
+ ** 0 of 186 failed
+
+VERIFICATION:- SUCCESSFUL
+```
+
+## `#[kani::solver(<solver>)]`
+
+**Changes the solver to be used by Kani.**
+
+This may change the verification time required to verify a harness.
+
+At present, `<solver>` can be one of:
+ - `minisat` (default): [MiniSat](http://minisat.se/), a minimalistic open-source SAT solver.
+ - `cadical`: [CaDiCaL](https://github.com/arminbiere/cadical), an extensible open-source SAT solver.
```

Comment:
Extensible SAT solver is actually the title of the [original MiniSAT paper](https://link.springer.com/chapter/10.1007/978-3-540-24605-3_37), so I'm not sure if it's a good description of cadical. I think adding descriptions for the solvers is not necessary.

Reply:
Thanks for the reference! Yeah, I removed the descriptions, which were based on their `README.md` files.